panc/src/main/scripts/panlint/panlint.py

```diff
@@ -571,7 +572,9 @@ def main():
         print('\nProblem count per file:')
         print(filestats_table(problem_stats))
 
-    print('\n%d problems found in %d lines' % (problem_count, len(problem_lines)))
+    if args.summary:
+        print()
+        print('\n%d problems found in %d lines' % (problem_count, len(problem_lines)))
```

Comment:
I think we can move to python3?
```suggestion
        print(f"\n{problem_count} problems found in {len(problem_lines)} lines")
```